FluentTerminal.SystemTray/Services/WinPty/WinPtySession.cs

```diff
@@ -18,9 +18,12 @@ public class WinPtySession : ITerminalSession
         private TerminalsManager _terminalsManager;
         private Process _shellProcess;
         private BufferedReader _reader;
+        private bool _enableBuffer;
```

Comment:
I would rather scope this to conpty only, since we had no issues with wintpy and the buffer

Reply:
Since we are adding this option to profile settings, I believe that it should be independent on ConPTY / WinPTY. It's true that currently we're experiencing issues only with ConPTY, but it may happen that similar happens with WinPTY and particular tool.
Although I can disable this option for WinPTY, I believe that it would only introduce some confusion.